outlines/using-packages.md

```diff
@@ -0,0 +1,34 @@
+# Using packages
+
+1. Packages vs modules
+  - Packages are useful for sharing content between apps and users
+  - It's usually better to leverage someone else's work and use a package if it does what you need
+2. Npm vs Atmosphere
+  1. Npm packages:
+    - Originally designed for server-side
+    - Systems like webpack and browserify let you bundle them for the client
+    - Meteor's module system seamlessly lets you `import` them from the client.
+    - You can find them at npmjs.org
+  1. Atmosphere packages are custom built for Meteor
+    - The Meteor package system has some advantages
+    - Code can be targeted at client *or* server
+    - Packages can be run through the build tool (e.g. coffeescript, less packages)
+    - Packages can define build plugins!
+    - Packages are namespaced to the author
+    - Binary stuff?
+    - You can find them atmospherejs.com
+3. Installing and using Atmosphere packages
+  1. Install with `meteor add X:Y`
+  2. Installed to `.meteor/packages` -- resolved in `.meteor/versions`
+  3. `.meteor/versions` means repeatable builds (?)
+  4. Atmosphere packages can include npm dependencies, managed for you.
```

Comment:
This is a subtle point. I don't think its the *recommended* way. Especially not for server-side dependencies.